driver-core/src/main/java/com/datastax/driver/core/querybuilder/QueryBuilder.java

```diff
@@ -589,6 +589,35 @@ public static Assignment set(String name, Object value) {
     }
 
     /**
+     * Advanced "set" assignment of a value to a column or a
+     * {@link com.datastax.driver.core.UserType UDT} field.
+     * <p/>
+     * Accepted types for the {@code name} parameter are:
+     * <ol>
+     * <li>String: {@code set("name", value)} generates {@code name = value}
+     * (this is equivalent to {@link #set(String, Object)});</li>
+     * <li>{@link #raw(String)}: {@code set(raw("name"), value)}
+     * generates {@code name = value};</li>
+     * <li>{@link #column(String)}: {@code set(column("name"), value)}
+     * generates {@code "name" = value}.</li>
+     * <li>{@link #path(String...)}: {@code set(path("udt", "field"), value)}
+     * generates {@code name.field = value}.</li>
```

Comment:
I wonder if it's worth mentioning `column`. I don't see any reason why you would want to use it here, and it's more confusing than helpful.